nodestream_akamai/edns/edns.py

```diff
@@ -1,57 +1,65 @@
+import asyncio
 import logging
-from ipaddress import IPv4Address, ip_address
 
 from nodestream.pipeline.extractors import Extractor
 
+from ..akamai_utils import addresses
 from ..akamai_utils.edns_client import AkamaiEdnsClient
 
+SUPPORTED_RECORD_TYPES = [
+    "A",
+    "AAAA",
+    "CNAME",
+    "NS",
+    "CAA",
+]
+
```

Comment:
Moved these out to both make them easier to see and also to prevent accidental instantiation of this list for every instance of AkamaiEdnsExtractor.